src/subaction_animations.jl

```diff
@@ -289,3 +289,55 @@ function _sethue(video, action, subaction, rel_frame)
     color = get_interpolation(subaction, rel_frame)
     Luxor.sethue(color)
 end
+
+"""
+    follow_path(points::Vector{Point}; closed=true)
+
+Can be applied inside a subaction such that the object defined in the parent action follows a path.
+It takes a vector of point which can be created as an example by calling `circle(O, 50)` <- notice that the action is set to `:none` the default.
+
+# Example
+```julia
+SubAction(1:150, follow_path(star(O, 300)))
+```
+
+# Arguments
+- `points::Vector{Point}` - the vector of points the object should follow
+
+# Keywords
+- `closed::Bool` default: true, sets whether the path is a closed path as for example in a circle, ellipse or any polygon
+"""
+function follow_path(points::Vector{Point}; closed = true)
+    (video, action, subaction, rel_frame) ->
+        _follow_path(video, action, subaction, rel_frame, points; closed = closed)
+end
+
+function _follow_path(video, action, subaction, rel_frame, points; closed = closed)
+    t = get_interpolation(subaction, rel_frame)
+    t -= floor(t)
+    if isapprox(t, 1.0, atol = 1e-4)
```

Comment:
I should check whether `t` is an integer first as when `t == 1` I actually want it to be `1` not `0`.
Additionally I want to do the `t -= floor(t)` thing only if `closed` as this otherwise doesn't make sense and should probably throw an error